src/components/profile/ProfileTeamLocation.vue

```diff
@@ -59,10 +60,25 @@ export default {
       return 'officeLocation:"' + this.officeLocation + '"'; // eslint-disable-line
     },
     timezoneWithTime() {
+      let dt = DateTime.local();
+      dt = dt.setZone(this.timezone);
+      const profileOffset = dt.offset;
+      dt = dt.setZone(this.currentTimezone);
+      const currentLocalOffset = dt.offset;
+      const printedOffsetRaw = this.getOffsetDiffFromMinutes(
+        currentLocalOffset,
+        profileOffset,
+      );
+      let printedOffset = '';
+      if (printedOffsetRaw !== null && printedOffsetRaw !== 0) {
+        printedOffset = ` | ${
+          printedOffsetRaw > 0 ? '+' : ''
+        }${printedOffsetRaw}hrs to you`;
```

Comment:
This renders offset to Nepal like _12.75hrs to you_, could we format it like _12:45hrs_ or _12hrs 45mins_ @mbransn any opinion?